### ISSUE TITLE
Scores for keypoints?

### ISSUE THREAD

**Issue by hnyll** [Parskatt/DeDoDe#25]
How to get scores for keypoints?

**Comment by Parskatt** [comment]
Perhaps this [demo](https://github.com/Parskatt/DeDoDe/blob/main/demo/demo_scoremap.py) is helpful?

This gives you unnormalized logits. We typically do a softmax after this stage, as can be seen [here](https://github.com/Parskatt/DeDoDe/blob/main/DeDoDe/detectors/dedode_detector.py#L46)
